### PR TITLE
Add guide to connect Segment to Inspector

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…t-inspector-to-segment**](https://docs-4hy137xbm.avo-dev.app/implementation/connect-inspector-to-segment) |
|:-:|
|<a href="https://docs-4hy137xbm.avo-dev.app/implementation/connect-inspector-to-segment">Screenshot of /implementation/connect-inspector-to-segment</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/mrjU98q-qMpSswY0hmYxmbyTEYaDL3gCZW05pwufiQaWEHdqnuZBfC7fEDFcSBCyqCqsoBKej5Yu3XoEg6JvNQ.png">(view full size)</a> |







Commit 196d2c2028fcbe74d17f64d031fae8328544783f (https://docs-4hy137xbm.avo-dev.app).

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+layout: 'guide'
+title: 'Send data from Segment to Avo Inspector (no code needed)'
+---
+
+import Link from '../../components/Link';
+
+# Enabling Avo Inspector without code changes
+
+## Use Avo Inspector as Segment destination
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:10]: Having these two titles in succession makes the top of the page a bit heavy. 

Screenshot 2021-05-27 at 15 09 58


Could we use the `abstract` field in the mdx header for either one of these titles?

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    src={require('../../images/chart-segment-to-inspector.png')}
+    alt="Data flow from Segment to Avo Inspector"
+  />
+</center>
+
+## Diagram of the data flow when sending to Segment with Avo generated code
+
+<center>
+  <img
+    height="500"
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:40]: This image looks skewed on my screen. Does removing the `height` attribute fix it?

**Review by hugihlynsson — COMMENTED** [review]
Just a quick review, will do a more thorough one later

**Review by bjornj12 — APPROVED** [review]
Really nice docs, like how step by step it is and good images. 
Feels trivial to follow this, the flow images are also great!
Good job @alex-tpom6oh 

Maybe a second ping on @hugihlynsson for his review, as he did not approve earlier

File: components/Navigation.tsx [hunk 3]
```diff
@@ -243,6 +243,12 @@ const navigation: navigationItem[] = [
         title: 'Using the Inspector SDK',
         path: '/implementation/setup-inspector-sdk',
       },
+      {
+        type: 'route',
+        title:
+          'Stream data from Segment to Inspector (no code changes required)',
```
[the remark this answers is not in the document]
Comment (hugihlynsson) [reply components/Navigation.tsx:249]: Another thing: Should this be in the _Implementation_  section? Would be be better to have it under tracking-plan/inspector/connect-segment?

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+</center>
+
+## Diagram of the data flow when sending to Segment with Avo generated code
+
+<center>
+  <img
+    src={require('../../images/chart-avo-to-segment-to-inspector.png')}
+    alt="Data flow from Segment to Avo Inspector with Avo Functions"
+  />
+</center>
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:38]: I feel like it's confusing to have both of these charts. I understand that the one above is for the connection and the lower one is the "old" way, using the Inspector SDK? Should we focus on the one that's related to this page?

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 5]
```diff
@@ -0,0 +1,222 @@
+---
+layout: 'guide'
+title: 'Send data from Segment to Avo Inspector (no code needed)'
+---
+
+import Link from '../../components/Link';
+
+## Use Avo Inspector as a Segment destination
+
+If you are sending your data to Segment you can stream it to Avo Inspector without adding any code to your codebase.
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:10]:
```suggestion
If you are already sending your data to Segment you can stream it to Avo Inspector without adding any code to your codebase.
```

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+
+<center>
+  <img
+    width="400"
+    src={require('../../images/guide-segment-to-inspector/inspect-sources.png')}
+    alt="Navigation to the Inspector sources screen in Avo"
+  />
+</center>
+
+If you already have the source, pick it. If you don’t have the source corresponding to your app, add a new one.
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:58]: Should we link to the connections docs here?

File: pages/implementation/connect-inspector-to-segment.mdx [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+</center>
+
+On the opened screen you’ll see options to integrate Inspector. Pick Middleware and Segment to see the integration code snippets.
+
+<center>
+  <img
+    height="400"
+    src={require('../../images/guide-segment-to-inspector/source-setup.png')}
+    alt="Source settings in Avo"
+  />
```
Comment (hugihlynsson) [inline pages/implementation/connect-inspector-to-segment.mdx:74]: Should we update this screenshot with this new UI?

**Review by hugihlynsson — APPROVED** [review]
Few suggestions but otherwise 🚀

File: pages/implementation/connect-inspector-to-segment.mdx (hunk 4 above)
Comment (alex-tpom6oh) [reply pages/implementation/connect-inspector-to-segment.mdx:38]: Both are about this new approach, but one is when you do not use Avo Functions and the other one is when you do. This is complicated to figure out even with the schemas 😅

Maybe needs a better explanation?

File: components/Navigation.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (alex-tpom6oh) [reply components/Navigation.tsx:249]: Good question 🤔

[the remark this answers is not in the document]
Comment (alex-tpom6oh) [reply components/Navigation.tsx:249]: Yeah I think it's more practical

File: pages/implementation/connect-inspector-to-segment.mdx (hunk 6 above)
Comment (alex-tpom6oh) [reply pages/implementation/connect-inspector-to-segment.mdx:58]: 👍

File: pages/implementation/connect-inspector-to-segment.mdx (hunk 7 above)
Comment (alex-tpom6oh) [reply pages/implementation/connect-inspector-to-segment.mdx]: updated

File: pages/implementation/connect-inspector-to-segment.mdx (hunk 4 above)
Comment (alex-tpom6oh) [reply pages/implementation/connect-inspector-to-segment.mdx]: moved the more complex diagram to the end and added some story about both

**Pull request merged**